presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+* Fix incorrect results when dividing certain decimal numbers.
+* Fix query failure for certain join queries with "Equi criteria are empty,
+  so INNER join should not have PARTITIONED distribution type".
+* Add support for recoverable grouped execution. This can be enabled with the
+  ``recoverable_grouped_execution`` session property.
+* Add support for grouped execution for ScanFilterProject-only queries. This
+  can be enabled with the ``grouped_execution_for_eligible_table_scans`` session
+  property.
+* Add config property ``max-concurrent-materializations`` and session property
+  ``max_concurrent_materializations`` to limit the number of plan sections that
+  will run concurrently when using materialized exchanges.
+* Add support for computing :func:`approx_distinct` over BingTile values.
+* Add :func:`merge_hll` which performs a scalar merge of an array of HyperLogLogs.
+* Add bitwise shift operations, :func:`bitwise_arithmetic_shift_right`,
+  :func:`bitwise_logical_shift_right` and :func:`bitwise_shift_left`.
+* Add completed and total lifespans on Web UI.
+
+Hive Connector Changes
+----------------------
+* Fix failures for ``information_schema`` queries when a table has an invalid storage format.
+* Improve query execution time over bucketed table with large buckets.
+* Add config property ``hive.metastore.glue.catalogid`` to configure the Glue catalog ID.
+
+Bytecode Changes
+----------------
+* Restore bytecode library deleted in version 0.193.
+
+SPI Changes
+-----------
+* Add experimental interface ``ConnectorPlanOptimizer`` to allow connectors to
+  participate in query plan optimization (e.g., filter pushdown). The interface
+  exposes part of the query plan to the connector to rewrite. The optimized plan
+  will be accepted by the engine in execution. This releases allows connectors
```

Comment:
@highker can you explain what does "The optimized plan will be accepted by the engine in execution" mean? Did you mean "The optimized plan will be accepted by the engine during execution" ?

Reply:
meaning "The optimized plan will be accepted by the engine during planning"

Reply:
"This releases" -> "This release"